core/tests/protractor_utils/forms.js

```diff
@@ -141,9 +142,13 @@ var ListEditor = function(elem) {
     }
   };
   var deleteItem = async function(index) {
-    await elem.element(
-      await by.repeater('item in localValue track by $index').row(index)
-    ).element(deleteListEntryLocator).click();
+    var deleteItemField = await by
+      .repeater('item in localValue track by $index')
+      .row(index);
+    var deleteItemFieldElem = elem.element(
```

Comment:
I think it should be `var deleteItemFieldElem = deleteItemField.element(deleteListEntryLocator);`. If not, then why did we declare `var deleteItemField`?

Reply:
done!